src/__tests__/element-queries.js

```diff
@@ -1182,3 +1182,65 @@ test('return a proper error message when no label is found and there is an aria-
 </div>"
 `)
 })
+
+// https://github.com/testing-library/dom-testing-library/issues/723
+it('gets form controls by label text on IE and other legacy browsers', () => {
+  // Simulate lack of support for HTMLInputElement.prototype.labels
+  const inputLabelsSpy = jest
+    .spyOn(HTMLInputElement.prototype, 'labels', 'get')
+    .mockReturnValue(undefined)
+
+  const {getByLabelText} = renderIntoDocument(`
```

Comment:
I'm not sure how necessary it is to copy/paste this from the other test. I think a single label that situation that covers the code path we're testing should suffice.

Reply:
Ideally we'd use the same test that is run with a test matrix covering different environments i.e. run with 
- `.labels` and `.control` (modern browser
- no `.labels` but `.control` (IE 11)
- neither `.lables` nor `.control` (edge 17)

I guess we can set something up like this in another PR where we leverage different jest environments that mock out problematic DOM methods we're aware of.

Reply:
> neither .lables nor .control (edge 17)

Shoot, I overlooked this. Appreciate you calling it out, @eps1lon ! 

I'll push my changes that reduce this to one test case for now. I also like the idea of running a comprehensive set of tests in different environments, and if it's preferable to take that approach instead of what's in this PR, please feel free to close this.

Reply:
Nah, let's get this in then we can add that in a future PR.

Reply:
Sure thing. Then if there are no other comments, this just needs an approval. 🙏